dev/com.ibm.ws.tx.jta_fat/publish/features/txjtafat-2.0.mf

```diff
@@ -2,7 +2,7 @@ Subsystem-ManifestVersion: 1
 IBM-Test-Feature: true
 Subsystem-SymbolicName: txjtafat-2.0; visibility:=public
 Subsystem-Version: 1.0.0
-Subsystem-Content: com.ibm.websphere.appserver.jta-2.0; type="osgi.subsystem.feature"
+Subsystem-Content: io.openliberty.jta-2.0; type="osgi.subsystem.feature"
 Subsystem-Type: osgi.subsystem.feature
 IBM-Feature-Version: 2
 IBM-API-Package: javax.transaction; type=spec,
```

Comment:
Should `javax.transaction` be `jakarta.transaction`, like it is in io.openliberty.jta-2.0?

Reply:
Yeah, this change is valid, though vestigial. It will not effect the jta and transaction fats as the transaction APIs are made visible by the dependency on jta-2.0.  I left the header, modified of course, to keep the feature structure uniform across all tx test features.  Thanks, Tom.